packages/env-utils/src/envUtils.native.ts

```diff
@@ -73,6 +73,15 @@ const getOsNameWeb = () => '';
 
 const getOsFamily = (): 'Linux' => 'Linux';
 
+/*
+ * On the app launch and then about once a minute (MESSAGES_FETCH_INTERVAL), a new config tries to fetch.
+ * MESSAGES_CHECK_INTERVAL is interval for checking if MESSAGES_FETCH_INTERVAL already expired.
+ * in miliseconds
+ */
+
+const MESSAGES_FETCH_INTERVAL = 600_000;
```

Comment:
Please always suffix name with units like `IN_MS` `IN_S` etc.

Reply:
I have followed previous naming convention. I'll update it

Reply:
3e7398770